.github/workflows/label-checker.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Label Checker
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - labeled
+      - unlabeled
+
+
+jobs:
+
+  check_labels:
+    name: Check labels
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Labels and Add Comment
+        id: check_labels_and_comment
+        uses: actions/github-script@v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            // Get PR info: id and labels
+            const prNumber = context.payload.pull_request.number;
+            const labels = context.payload.pull_request.labels.map(label => label.name);
+
+            // If no descriptive label is set, add a comment in the PR and make the action check fail
+            if (!labels.includes('enhancement') && !labels.includes('deprecated') && !labels.includes('refactoring') && !labels.includes('pr: breaking') && !labels.includes('pr: clean') && !labels.includes('pr: fix') && !labels.includes('pr: new feature') && !labels.includes('pr: test')) {
```

Comment:
I would create an array of stuff to compare with, it will be easier to add/remove tags in the future